messages/src/client.rs

```diff
@@ -88,17 +90,13 @@ impl Hoverkite {
         Ok(())
     }
 
-    /// Makes the buzzer play the given frequency until otherwise instructed.
-    pub fn set_buzzer_frequency(&mut self, frequency: Option<u32>) -> Result<(), io::Error> {
-        let command = Command::SetBuzzerFrequency(frequency.unwrap_or(0));
-        self.send_command(Side::Left, command)
-    }
-
     /// Plays the given sequence of notes on the hoverboard.
     pub fn play_notes(&mut self, notes: &[Note]) -> Result<(), io::Error> {
         for note in notes {
+            trace!("Sending {:?}", note);
             let command = Command::AddBuzzerNote(*note);
             self.send_command(Side::Left, command)?;
+            sleep(NOTE_SEND_SLEEP_DURATION);
```

Comment:
:/

can we rename this function to `play_notes_blocking()` or something suitably scary, to make it obvious that you don't want to include it in your control loop?

Reply:
Done.